NEWS.md

```diff
@@ -5,6 +5,7 @@
 * [MODSOURCE-773](https://folio-org.atlassian.net/browse/MODSOURCE-773) MARC Search omits suppressed from discovery records in default search
 * [MODINV-1044](https://folio-org.atlassian.net/browse/MODINV-1044) Additional Requirements - Update Data Import logic to normalize OCLC 035 values
 * [MODSOURMAN-1200](https://folio-org.atlassian.net/browse/MODSOURMAN-1200) Find record by match id on update generation 
+* [MODSOURCE-785](https://folio-org.atlassian.net/browse/MODSOURCE-785) Find record by match id on update generation
```

Comment:
```suggestion
* [MODSOURCE-785](https://folio-org.atlassian.net/browse/MODSOURCE-785) Update 005 field when set MARC for deletion
```

Reply:
@RuslanLavrov Good point! It was a TYPO. Thank you!